### DISCUSSION TITLE
xcodebuild: WARNING: Using the first of multiple matching destinations:

### DISCUSSION THREAD

**Discussion by walkthelot** [appcelerator/titanium_mobile#13174] [Q&A] [answered]
Hello all. I am at my wits end. Trying to get TI development environment setup on a new Mac Mini M1. I've setup Atom and VSCode to successfully install on iOS and Android Simulators and also Android devices. No matter what I try I can not get it to install on any iOS devices. I've checked my certificates and provisioning profiles time and time again and all is good. I am able to install to device from XCode (created a test project).

The iOS devices show up as targets in Atom and VSCode and also via "appc info". In appc info everything looks good.
Any help would be GREATLY appreciated. I am like a week and half in and 5 virgin MacOS installs trying to get this going.

Here is what I get in console upon building to install on iOS device:
[INFO]   Invoking xcodebuild
[ERROR]  2021-11-11 19:46:05.864 xcodebuild[4928:34077] DVTAssertions: Warning in /Library/Caches/com.apple.xbs/Sources/DVTiOSFrameworks/DVTiOSFrameworks-19450/DTDeviceKitBase/DTDKRemoteDeviceData.m:373
[ERROR]  Details: (null) deviceType from 00008020-000C19CC2E69402E was NULL when -platform called.
[ERROR]  Object: <DTDKMobileDeviceToken: 0x7fe07ac56520>
[ERROR]  Method: -platform
[ERROR]  Thread: <NSThread: 0x600002b1c180>{number = 3, name = (null)}
[ERROR]  Please file a bug at https://feedbackassistant.apple.com with this warning message and any useful information you can provide.
[ERROR]  --- xcodebuild: WARNING: Using the first of multiple matching destinations:

MacOS 12.0.1 (latest public release)
XCode 13.1 (latest public release)
appc version 9.0.1 (latest I believe)
appc ti 5.3.2
TI SDK 10.1.0.GA
JDK 14.0.2
Node.js 14.18.1

**Comment by cb1kenobi** [comment]
That’s a pretty interesting error. I thought the destination stuff related to simulator builds, not device builds.

Just out of curiosity, which Titanium SDK version and which Xcode version are you using? Have you tried a different SDK version?

**Reply by walkthelot** [reply]
I'm going to try a prior and beta Xcode version as soon as I finish a couple of Time Machine backups. Didn't seem to change anything trying different TI SDK versions but I will try again since my head has been going in circles :). I wonder if Apple made a change to how things are called or have a bug in XCode. 

PS: Tried on iPad running iPadOS 15 (latest) and also an old iPhone 6s running iOS 14.x

MacOS 12.0.1 (latest public release)
XCode 13.1 (latest public release)
appc version 9.0.1 (latest I believe)
appc ti 5.3.2
TI SDK 10.1.0.GA
JDK 14.0.2

**Reply by walkthelot** [reply]
https://github.com/appcelerator/titanium_mobile/issues/13175#issue-1051548916

**Reply by walkthelot** [reply]
I'm pretty sure I tried different SDK versions but at this point I've been in so many circles... LOL. Tomorrow I'll try difference Xcode versions, SDK versions, JDK versions, Node versions... just gonna try different of them all I think.

**Reply by m1ga** [reply]
can you make this change in your local file: https://github.com/appcelerator/titanium_mobile/pull/13172/files

**Comment by ewanharris** [comment, marked as answer]
This was fixed in #13098 and backported to 10.1.1.GA
